File: src/library/Shift.Sdk/Service/Platform/DashboardNotifications/ActiveNotification.cs
using System;

namespace Shift.Sdk.Service.Platform.DashboardNotifications
{
    public class ActiveNotification
    {
        public Guid NotificationId { get; set; }
        public DashboardNotificationType Type { get; set; }
        public string Title { get; set; }
        public string Details { get; set; }
        public string LinkText { get; set; }
        public string LinkUrl { get; set; }
        public DateTimeOffset Modified { get; set; }
    }
}

File: src/ui/Shift.UI/src/routes/admin/home/ActiveNotification.ts
import { ApiDashboardNotificationType } from "@/api/controllers/platform/dashboard/ApiDashboardNotificationType";
import { DateTimeParts } from "@/helpers/date/dateTimeTypes";

export interface ActiveNotification {
    notificationId: string;
    type: ApiDashboardNotificationType;
    title: string;
    details: string | null;
    linkText: string | null;
    linkUrl: string | null;
    modified: DateTimeParts;
}